src/Compilers/CSharp/Test/EndToEnd/EndToEndTests.cs

```diff
@@ -327,24 +327,25 @@ static void runTest(int n)
         public void LongInitializerList()
         {
             CancellationTokenSource cts = new CancellationTokenSource(TimeSpan.FromSeconds(30));
+            int iterations = 0;
             try
             {
-                initializerTest(cts.Token);
+                initializerTest(cts.Token, ref iterations);
             }
-            catch (TaskCanceledException)
+            catch (Exception e) when (e is OperationCanceledException or TaskCanceledException)
```

Comment:
nit: it looks like 'OperationCanceledException' is the base type of 'TaskCanceledException'. So this could maybe be simplified.